### PR TITLE
Integrated user's name in dashboard with email

### LINKED ISSUE #146: [FRONTEND] Integrated user's name shown in Dashboard
Integrated user's name shown in Dashboard when logged in.

### PR DESCRIPTION
**Pull request by Anirban-1490**
Fixes #146 

Added user's name and additionally email in the dashboard after login

Image - 

Capture

### CODE REVIEW

**Comment by BRAVO68WEB** [comment]
Can you make the "Brand Name" to URL Minify

**Review by BRAVO68WEB — APPROVED** [review]

File: frontend/helpers/user/userState.js [hunk 1]
```diff
@@ -49,13 +49,13 @@ const UserAuthProvider = ({ children }) => {
...
   }
-  const createAcc = async ({ email, password, name }) => {
+  const createAcc = async ({ email, password, username }) => {
     let register = true
     await axios
       .post(`/user/register`, {
         email: email,
         password: password,
-        name: name,
+        name: username,
```
Comment (BRAVO68WEB) [inline frontend/helpers/user/userState.js:58]: Why is this "username" as you have previously mentioned it as "name"

**Review by BRAVO68WEB — CHANGES_REQUESTED** [review]

**Comment by Anirban-1490** [comment]
> @BRAVO68WEB Check it now, did all the changes you asked .
> 
> Capture

@BRAVO68WEB ?

**Comment by BRAVO68WEB** [comment]
Ok I will. Heck by tomorrow and merge

**Review by BRAVO68WEB — APPROVED** [review]

**Pull request merged**